cmd/mattermost/commands/server.go

```diff
@@ -97,6 +104,17 @@ func runServer(configStore config.Store, disableConfigWatch bool, usedPlatform b
 	return nil
 }
 
+func startTelemetryServer(telemetryAddress *string, logger *log.Logger) {
+	logger.Println("Telemetry Server starting.")
+	clientMetricServer := &http.Server{
+		Addr:     *telemetryAddress,
+		Handler:  promhttp.Handler(),
+		ErrorLog: logger,
+	}
+
+	logger.Println("Telemetry Server stopped.", clientMetricServer.ListenAndServe())
```

Comment:
is it intended to be Telemetry sever initialized?

Reply:
No When server stops then only ListenAndServe will return.